packages/monaco-editor/src/completions/completionItemProvider.ts

```diff
@@ -0,0 +1,243 @@
+import * as monaco from "monaco-editor";
+import { Observable, Observer } from "rxjs";
+import { first, map } from "rxjs/operators";
+import { childOf, JupyterMessage, ofMessageType, Channels } from "@nteract/messaging";
+
+/**
+ * TODO: import from nteract when the changes under editor-base.ts are ported to nteract.
```

Comment:
Is this something that should be addressed with this check in or is this a todo task for the future?